entity/post.go

```diff
@@ -0,0 +1,19 @@
+package entity
+
+import (
+	"gorm.io/gorm"
+	"time"
+)
+
+type Post struct {
+	gorm.Model
+	Title       string `gorm:"type:VARCHAR(64)"`
+	Content     string `gorm:"type:VARCHAR(2048)"`
+	CandidateID uint
```

Comment:
same as [previous](https://github.com/bccfilkom-be/freepass-2024/pull/5/files#r1485627211) comment